community/resources/docs-as-code.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: full-width
+group: community
+title: Docs as Code
+---
+
+To further expand documentation options for DevDocs, we offer a new opportunity for developers to contribute content through Community Engineering project repositories!
+
+Docs as Code is a new process that pulls content from public GitHub repositories and generates it as part of DevDocs. The DevDocs team helps you write add your content for the site. 
```

Comment:
This isn't _technically_ accurate. We use a "[docs as code](http://www.writethedocs.org/guide/docs-as-code/)" approach to manage _all_ of our developer-facing content. This isn't a new concept. Although we _are_ introducing a new concept for contributing docs, we shouldn't use a term that means something else to refer to that new concept.

Reply:
Gotcha, good point, renaming to Remote Doc Repo for now. Thoughts on that renaming? That name would cover remote doc directory and remote Jekyll projects.

Reply:
We've been using "multi-repo devdocs" and "multi-source devdocs" for internal planning. I think either of those conveys the meaning well.